dashboard/app/helpers/application_helper.rb

```diff
@@ -111,16 +111,15 @@ def external_oauth_sign_out_url(provider)
     end
   end
 
-  def meta_image_url(params)
-    level_source = params[:level_source]
-    if level_source
-      app = level_source.level.game.app
-    else
-      app = params[:app]
+  def meta_image_url
```

Comment:
we often encounter bugs related to expecting instance variables in our views, so yeah, I'm beginning to prefer standardizing on argument-passing instead of instance variables in the controller layer, so the inputs-outputs are more explicit and more easily understood (and easier to refactor).